### DISCUSSION TITLE
Question: How are post URLs determined?

### DISCUSSION THREAD

**Discussion by ElectricRCAircraftGuy** [mmistakes/minimal-mistakes#2727] [Q&A] [answered]
With the minima theme (https://github.com/jekyll/minima), anything in my `_posts` dir named `2019-09-19-ubuntu-18-post-install-steps.md` automatically gets placed at the URL `/2019/09/19/ubuntu-18-post-install-steps.html`. However, with the minimal-mistakes theme it seems to go to `/ubuntu-18-post-install-steps/` instead. This means I need to set up redirects (https://github.com/mmistakes/minimal-mistakes/issues/551#issuecomment-249037145) as I migrate to your theme.

1. How does the automatic URL get set? 
    1. Can we change this if we like? How?
1. What advantages does it have over the `minima` style which is categorized by year/month/day, and ends in `.html` instead of just `/`?

**Comment by iBug** [comment, marked as answer]
Minima just sets it to some kind of "default" style and this is not theme-specific.

You can always set your permalink style in [`_config.yml`](https://github.com/mmistakes/minimal-mistakes/blob/a0309bba05161ee9968ec9601604aedb91b756e3/_config.yml#L209). Check the [Jekyll documentation](https://jekyllrb.com/docs/permalinks/) for more information.
